### PR TITLE
spring GET/POST and jpa relations

### PR DESCRIPTION
**Pull request by rkrylov**

### CODE REVIEW

**Comment by TrueJugernaut** [comment]
it works! LGFM

File: src/main/java/mate/academy/springdemo/Main.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import mate.academy.springdemo.service.DeveloperService;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+public class Main {
+    public static void main(String[] args) {
+        ApplicationContext context = new AnnotationConfigApplicationContext(Config.class);
+
+        DeveloperService developerService = context.getBean(DeveloperService.class);
+        Developer dev2 = Developer.builder().id(1L).name("Vasya").age(30).salary(900).build();
```
Comment (boroda4436) [inline src/main/java/mate/academy/springdemo/Main.java]: @Orbit419  @TrueJugernaut  it's not ok to insert id manually. It should be auto generated

File: src/main/java/mate/academy/springdemo/Main.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        developerService.create(dev2);
```
Comment (boroda4436) [inline src/main/java/mate/academy/springdemo/Main.java]: @Orbit419 @TrueJugernaut  we don't using Main class. We are using Tomcat to launch the app

File: src/main/java/mate/academy/springdemo/controller/DeveloperController.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+    @RequestMapping(value = "/newDev", method = RequestMethod.GET)
+    public ModelAndView newDeveloper() {
+        return new ModelAndView("developers/createDeveloper", "developer", new Developer());
+    }
+
+    @RequestMapping(value = "/addDev", method = RequestMethod.POST)
+    public String putDeveloper(@ModelAttribute("developer") DeveloperDto devDto, ModelMap model) {
+        Developer dev = Developer.builder()
+                .id(devDto.getId())
```
Comment (boroda4436) [inline src/main/java/mate/academy/springdemo/controller/DeveloperController.java]: @Orbit419  @TrueJugernaut id should be auto generated

File: src/main/resources/database.properties [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+db.driver=com.mysql.cj.jdbc.Driver
+db.url=jdbc:mysql://localhost:3306/developers_spring?serverTimezone=UTC
+db.user=root
+db.password=1234
+
+#hibernate config
+db.hibernate.dialect=org.hibernate.dialect.MySQLDialect
+db.hibernate.show_sql=true
+db.entitymanager.packages.to.scan=mate.academy.springdemo.model
+hibernate.hbm2ddl.auto=update
```
Comment (boroda4436) [inline src/main/resources/database.properties]: empty line is required at the EOF

File: src/main/webapp/WEB-INF/pages/developers/info.jsp [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+   <title>Developer Info</title>
+   <%--<link rel="stylesheet" type="text/css" href="styles/style.css">--%>
+</head>
+<body>
+    <h1>
+        Hello, I'm developer!
+    </h1>
+</body>
+</html>
```
Comment (boroda4436) [inline src/main/webapp/WEB-INF/pages/developers/info.jsp]: empty line is required at the EOF

File: src/main/webapp/WEB-INF/pages/developers/id.jsp [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+  User: Руслан
+  Date: 27.01.2019
+  Time: 11:42
+  To change this template use File | Settings | File Templates.
+--%>
+<jsp:useBean id="developer" scope="request" type="mate.academy.springdemo.model.dto.DeveloperDto"/>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Developer By Id</title>
```
Comment (boroda4436) [inline src/main/webapp/WEB-INF/pages/developers/id.jsp]: it will be better to name this jsp like info.jsp

**Review by boroda4436 — CHANGES_REQUESTED** [review]

**Comment by boroda4436** [comment]
@TrueJugernaut don't forget to approve the PR if you are sure that everything is ok

**Review by boroda4436 — APPROVED** [review]
LGTM

**Pull request merged**